README.md

```diff
@@ -158,13 +158,13 @@ main();
 
 ### Error Reporting
 
-Any `Error` objects you log at severity `error` or higher can automatically be picked up by [Stackdriver Error Reporting](https://cloud.google.com/error-reporting/) if you have specified a `serviceContext.service` when instantiating a `LoggingWinston` instance:
+Any `Error` objects you log at severity `error` or higher can automatically be picked up by [Error Reporting](https://cloud.google.com/error-reporting/) if you have specified a `serviceContext.service` when instantiating a `LoggingWinston` instance:
```

Comment:
README is autogenerated in this repo (see [this comment](https://github.com/googleapis/nodejs-logging-winston/pull/676#issuecomment-1071935488)). Use .readme-partials.yaml to modify the content. If the content is shared among multiple repos then it should be modified in https://github.com/googleapis/synthtool/tree/master/synthtool/gcp/templates/node_library

Reply:
The original change comes from [readme-partials.yaml](https://github.com/googleapis/nodejs-logging-winston/pull/676/files#diff-df78b4580f896b4bfa3eba25bc40c45ab1673a5a60f2cac25764a128a0a372af) and README reflecting here the change